src/assets/dotnet.md

```diff
@@ -1,5 +1,54 @@
 ## .NET
 ---
 ### Prerequisites
+In order to use Allure in your .Net project with the following exemple you will need used NUnit with NuGet NUnit.Allure
 
 ### Writing a Test case
+The tag [AllureNUnit] its necessary to generate the JSON files.
+In this exemple, the [Description] tag is set as step test case in the allure-reporter tool.  
+The [AllureTms] tag contains the test case name in Jama.
+
+Use the following example as a basis for your test cases.
+
+```C#
+    [TestFixture("ACR", "mg/mmol")]
+    [TestFixture("Chol/HDL", "")]
+    [AllureNUnit]
+    public class QCAfinionTestCases : UnitTestBase<AfinionAnalyzer>
+    {
+        string code;
+        string unit;
+        string flag;
+
+        XDocument translatedResult = null;
+        OBSR01 observationMessage = null;
+        HELR01 helloMessage = null;
+
+        public QCAfinionTestCases(string code, string unit)
+        {
+            this.code = code;
+            this.unit = unit;
+        }
+
+        [AllureEpic("TestAfinion")]
+        [AllureTms("When the device has a control result for upload, Then the message shall be upload to GWP")]
+        [AllureFeature("SYN-TC-2643 - The scope of this scenario is to verify that the translator returns the message as expected. QC results are shown in GEM Web Plus.")]
+        [Description("Check that the mapping for ACR is correct.")]
+        [Test]
+        public void MessageTranslated_As_Expected()
+        {
+            HELR01 helloMessage = this.Analyzer.SendHelloMessage();
+            this.Analyzer.SendStatusMessage(newObservations: 1);
+            OBSR02 observationMessage = this.Analyzer.SendControlResultsMessage(
+                this.Analyzer.MessageBuilder.WithTest(code, unit));
+            this.Analyzer.WaitForTerminateMessage();
+            XDocument translatedResult = this.Gwp.WaitForResults(1)?.Message;
+            translatedResult.Should().NotBeNull();
```

Comment:
Each step may have at least an expected result. Please, use the JSON tag [Step] to document this assertion.
You can use [Step]("Action: ...") to identify that a step is an Action. Otherwise, all the steps will be considered as Expected Results. Consider that it may be nested Steps. All the Expected Results are documented just once, for each type of object. Refer to the class TestUtil.java at https://github.com/systelab/seed-jee/blob/master/src/test/java/com/systelab/seed/utils/TestUtil.java

Reply:
For C# the Description] tag is the step test case beucause no implemented yet the Action and Expected, and the tags documented in git for NUnitAllure not runs.
For this, the description is the Action and the Expected is the testcase name in the code.